chapter_codecs.md

```diff
@@ -72,6 +72,84 @@ The one and only command existing for the moment is `GotoAndPlay( ChapterUID );`
 same suggests, it means that, when this command is encountered, the `Matroska Player`
 **SHOULD** jump to the `Chapter` specified by the UID and play it, as long as this `Chapter` exists.
 
+## MatroskaJS
+
+This is the case when `ChapProcessCodecID` = 2. This is a script language build for
+Matroska purposes. It uses the [@!ECMAScript] 6th Edition (ES6) syntax. The commands are stored as text commands, in UTF-8.
```

Comment:
We  may want to use ES5 instead.

Reply:
That would be better (less constraint) but we don't have a working solution with that.

Also there are [things](https://compat-table.github.io/compat-table/es6/) that could make the JS usage cleaner like [let and const](https://hacks.mozilla.org/2015/07/es6-in-depth-let-and-const/) or static arrays and [typed arrays](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/TypedArray). For example you could have a global function that gets the list of choices in an array and call the MatroskaJS engine. Then each (enter) chapter would just had the array and that call.